CHANGELOG

```diff
@@ -19,6 +19,9 @@ Yosys 0.9 .. Yosys 0.9-dev
     - Added automatic gzip compression (based on filename extension) for backends
     - Improve attribute and parameter encoding in JSON to avoid ambiguities between
       bit vectors and strings containing [01xz]*
+    - Added "clkbufmap" pass
+    - Added "synth_xilinx -ise" for Spartan 6 (experimental)
```

Comment:
I would just leave it as `synth_xilinx -ise` since it is not Spartan 6 specific.
You can add another entry that adds `synth_xilinx -family xc6s` for Spartan 6 (experimental)
I would also add entries for `-iopad/-noipad`